sql/flow.go

```diff
@@ -17,12 +17,12 @@ import (
 )
 
 const (
-	astroSQLCLIProjectURL     = "https://pypi.org/pypi/astro-sql-cli/json"
-	astroSQLCLIConfigURL      = "https://raw.githubusercontent.com/astronomer/astro-sdk/astro-cli/sql-cli/config/astro-cli.json"
-	SQLCliDockerfilePath      = ".Dockerfile.sql_cli"
-	SQLCLIDockerfileWriteMode = 0o600
-	SQLCliDockerImageName     = "sql_cli"
-	PythonVersion             = "3.9"
+	AstroSQLCLIProjectURL = "https://pypi.org/pypi/astro-sql-cli/json"
+	AstroSQLCLIConfigURL  = "https://raw.githubusercontent.com/astronomer/astro-sdk/1473-check-python-sdk-version/sql-cli/config/astro-cli.json"
```

Comment:
I am guessing this is temporary, leaving a comment for our reference to remove it before merging the PR